ui/admin.html

```diff
@@ -70,7 +69,7 @@
   <script src="vendor/angular/angular-cookies.js"></script>
   <script src="vendor/angular/angular-ui-router.js"></script>
   <script src="vendor/angular/angular-sanitize.js"></script>
-  <script src="vendor/angular-local-storage.min.js"></script>
+  <script src="vendor/ngStorage.min.js"></script>
```

Comment:
this is an easier package to use and we weren't using localstorage anywhere else, afaict.  But, yeah, I'll move it into another PR.